src/internal/EnhancedButton.js

```diff
@@ -37,31 +37,139 @@ function listenForTabPresses() {
 
 class EnhancedButton extends Component {
   static propTypes = {
+    /** @ignore */
     centerRipple: PropTypes.bool,
+
+    /** @ignore */
     children: PropTypes.node,
+
+    /** @ignore */
     containerElement: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.element,
     ]),
+
+    /** @ignore */
     disableFocusRipple: PropTypes.bool,
+
+    /** @ignore */
     disableKeyboardFocus: PropTypes.bool,
+
+    /** @ignore */
     disableTouchRipple: PropTypes.bool,
+
+    /** @ignore */
     disabled: PropTypes.bool,
+
+    /** @ignore */
     focusRippleColor: PropTypes.string,
+
+    /** @ignore */
     focusRippleOpacity: PropTypes.number,
+
+    /** @ignore */
     keyboardFocused: PropTypes.bool,
+
+    /** @ignore */
     linkButton: PropTypes.bool,
+
+    /**
+     * Callback function fired when the element loses focus.
+     * @param {object} event `blur` event targeting the element.
+     */
     onBlur: PropTypes.func,
+
+    /** @ignore */
     onClick: PropTypes.func,
+
+    /**
+     * Callback function fired when the element gains focus.
+     * @param {object} event `focus` event targeting the element.
+     */
     onFocus: PropTypes.func,
+
+    /**
+     * Callback function fired when a key is pressed.
+     * @param {object} event `keyboard` event targeting the element.
+     */
     onKeyDown: PropTypes.func,
+
+    /**
+     * Callback function fired when a key is released.
+     * @param {object} event `keyboard` event targeting the element.
+     */
     onKeyUp: PropTypes.func,
+
+    /**
+     * Callback function fired when the element is focused or blurred by the keyboard.
+     *
+     * @param {object} event `focus` or `blur` event targeting the element.
+     * @param {boolean} keyboardFocused Indicates whether the element is focused.
+     */
     onKeyboardFocus: PropTypes.func,
+
+    /**
+     * Callback function fired when a mouse button is pressed down on the element.
+     *
+     * @param {object} event `mousedown` event targeting the element.
+     */
+    onMouseDown: PropTypes.func,
+
+    /**
+     * Callback function fired when the mouse enters the element.
+     *
+     * @param {object} event `mouseenter` event targeting the element.
+     */
+    onMouseEnter: PropTypes.func,
+
+    /**
+     * Callback function fired when the mouse leaves the element.
+     *
+     * @param {object} event `mouseleave` event targeting the element.
+     */
+    onMouseLeave: PropTypes.func,
+
+    /**
+     * Callback function fired when a mouse button is released on the element.
+     *
+     * @param {object} event `mouseup` event targeting the element.
+     */
+    onMouseUp: PropTypes.func,
+
+    /**
+     * Callback function fired when a touchTap event ends.
+     *
+     * @param {object} event `touch` event targeting the element.
+     */
+    onTouchEnd: PropTypes.func,
```

Comment:
Yes, this was from before I suggested we remove the native event callbacks from the component docs. If all that's left is `onTouchTap`, it might as well go in the `Chip` prop docs rather than separately imported from `EnhancedButton`.